Assets/StreamingAssets/LUA/Furniture.lua

```diff
@@ -216,48 +226,48 @@ function Stockpile_UpdateAction(furniture, deltaTime)
     end
 
 
-    -- TODO: In the future, stockpiles -- rather than being a bunch of individual
-    -- 1x1 tiles -- should manifest themselves as single, large objects.  This
-    -- would respresent our first and probably only VARIABLE sized "furniture" --
-    -- at what happenes if there's a "hole" in our stockpile because we have an
-    -- actual piece of furniture (like a cooking stating) installed in the middle
-    -- of our stockpile?
-    -- In any case, once we implement "mega stockpiles", then the job-creation system
-    -- could be a lot smarter, in that even if the stockpile has some stuff in it, it
-    -- can also still be requestion different object types in its job creation.
+	-- TODO: In the future, stockpiles -- rather than being a bunch of individual
+	-- 1x1 tiles -- should manifest themselves as single, large objects.  This
+	-- would respresent our first and probably only VARIABLE sized "furniture" --
+	-- at what happenes if there's a "hole" in our stockpile because we have an
+	-- actual piece of furniture (like a cooking stating) installed in the middle
+	-- of our stockpile?
+	-- In any case, once we implement "mega stockpiles", then the job-creation system
+	-- could be a lot smarter, in that even if the stockpile has some stuff in it, it
+	-- can also still be requestion different object types in its job creation.
 
     local itemsDesired = {}
 
-    if (furniture.Tile.Inventory == nil) then
-        --ModUtils.ULog("Creating job for new stack.")
-        itemsDesired = Stockpile_GetItemsFromFilter( furniture )
-    else
-        --ModUtils.ULog("Creating job for existing stack.")
-        desInv = furniture.Tile.Inventory.Clone()
-        desInv.MaxStackSize = desInv.MaxStackSize - desInv.StackSize
-        desInv.StackSize = 0
+	if( furniture.Tile.Inventory == nil ) then
+		--ModUtils.ULog("Creating job for new stack.")
+		itemsDesired = Stockpile_GetItemsFromFilter( furniture )
+	else
+		--ModUtils.ULog("Creating job for existing stack.")
+		desInv = furniture.Tile.Inventory.Clone()
+		desInv.MaxStackSize = desInv.MaxStackSize - desInv.StackSize
+		desInv.StackSize = 0
 
         itemsDesired = { desInv }
     end
 
-    local job = Job.__new(
-        furniture.Tile,
-        nil,
-        nil,
-        0,
-        itemsDesired,
-        Job.JobPriority.Low,
-        false
-    )
-    job.JobDescription = "job_stockpile_moving_desc"
-    job.acceptsAny = true
+	local j = Job.__new(
```

Comment:
That is an older version, where I tried to fix all those things and made this PR a lot harder to review and work with. I am not going to refactor Lua in this PR, changing Lua is harder since you don't get errors, so it requires more testing.

Make an issue to settle on a style for Lua, since Lua is a mess. And then we can fix all the issues in 1 or multiple PR. I will be happy to fix them later.